doc/remotes.md

```diff
@@ -0,0 +1,52 @@
+# Remotes
+
+## Introduction
+
+Remotes (machine with LXD installed) is concept used by LXD through which users doesn't have to always provide hostname or IP and then validating certificate information.
+LXD introduce three types of remotes:
+
```

Comment:
Should this be:
 - Static (hardcoded)
 - Local (per-user)
 - Global (per-system)

Instead?